{{cookiecutter.github_project_name}}/MANIFEST.in

```diff
@@ -1,4 +1,5 @@
-recursive-include {{ cookiecutter.python_package_name  }}/static *.*
+recursive-include {{ cookiecutter.python_package_name  }}/nbextension *.*
```

Comment:
I've renamed the `static` dir into `nbextension`, I think it looks better now with the new `labextension` dir on its side.

Reply:
👍